bespin/option_spec/bespin_specs.py

```diff
@@ -302,8 +284,8 @@ def stack_spec(self):
             , stack_json = valid_stack_json(default="{config_root}/{_key_name_1}.json")
             , stack_yaml = valid_stack_yaml(default="{config_root}/{_key_name_1}.yaml")
 
-            , params_json = valid_params_json(default="{config_root}/{environment}/{_key_name_1}-params.json")
-            , params_yaml = valid_params_yaml(default="{config_root}/{environment}/{_key_name_1}-params.yaml")
```

Comment:
Details in PR and commit message. There's currently no situation where the default could be used due to the enforcement of one or the other. ie: the user must pass a value, and that value would be used over the NotSpecified one.
Change to these lines makes this clearer.